doc/adr/0002-summary-of-decisions-thus-far.md

```diff
@@ -0,0 +1,85 @@
+# 2. Summary of decisions thus far
+
+Date: 2024-10-09
+
+## Status
+
+Accepted
+
+## Context
+
+This is a high-level summary of previous decisions that have been made on this
+project. ADRs will be used for all new architectural decisions.
+
+## Decision
+
+Below is a list with the specific explanations on the architectural decisions
+that have been made thus far in this project.
+
+#### Use Django and Django Admin.
+
+Previously, a Web 1.0 Python application was being built to create this
+microservice. It was felt that utilizing Django and the built in Admin service
+could easily support the projected user base size of the Affiliations
+Microservice and it would increase the speed of delivery on this project. It was
+also a benefit that it was another Python framework and a tool being utilized on
+another project on the team to create cross over skills.
+
+Although it is not recommended to base your website solely on the built-in admin
+service, since this is going to be a small internal tool to replace a
```

Comment:
Although it is not recommended to base your website solely on the built-in admin
~~service~~ site

Reply:
Made this change!